packages/interface/src/config.ts

```diff
@@ -7,6 +7,21 @@ export const metadata = {
   image: "/api/og",
 };
 
+/**
+ * Get the PIMLICO RPC URL based on the network we are connected to
+ * @returns the PIMLICO RPC URL
+ */
+export const getPimlicoRPCURL = (): string | undefined => {
+  switch (process.env.NEXT_PUBLIC_CHAIN_NAME) {
+    case "optimism":
+      return `https://api.pimlico.io/v2/optimism/rpc?apikey=${process.env.NEXT_PUBLIC_PIMLICO_API_KEY}`;
+    case "optimismSepolia":
+      return `https://api.pimlico.io/v2/optimism-sepolia/rpc?apikey=${process.env.NEXT_PUBLIC_PIMLICO_API_KEY}`;
+    default:
+      return undefined;
+  }
+};
```

Comment:
it would still be visible in the network tab through http requests. The key will be protected with a cors check to be coming from the production domain